Wikipedia/Code/WKWebView+WMFWebViewControllerJavascript.swift

```diff
@@ -156,27 +156,30 @@ extension WKWebView {
         let readMoreItemCount = 3
         evaluateJavaScript("window.wmf.footerReadMore.add('\(title)', \(readMoreItemCount), 'pagelib_footer_container_readmore_pages', '\(proxyURL)', \(saveButtonTapHandler), \(titlesShownHandler), document);", completionHandler: nil)
     }
-    
-    public func wmf_enableCompatibilityTransformSupport(){
-        evaluateJavaScript("window.wmf.compatibility.enableSupport(document);", completionHandler: nil)
-    }
 
     public func wmf_classifyThemeElements(){
+        // 'themes.classifyElements()' needs to happen once after body elements are present. it
+        // classifies some tricky elements like math formula images (see 'enwiki > Quadradic formula')
         evaluateJavaScript("window.wmf.themes.classifyElements(document);", completionHandler: nil)
     }
     
-    public func wmf_applyTheme(_ theme: Theme){
+    static public func wmf_themeApplicationJavascript(with theme: Theme) -> String{
         var jsThemeConstant = "DEFAULT"
```

Comment:
This is practical and good as is but in case it wasn't clear, these constants are also exposed in JavaScript as `pagelib.ThemeTransform.THEME.DEFAULT` / `DARK` / `SEPIA`.